enterprise/frontend/src/embedding-sdk/components/private/SdkLicenseProblem/SdkLicenseProblemBanner.module.css

```diff
@@ -0,0 +1,34 @@
+.PopoverDropdown.PopoverDropdown {
+  border: none;
+}
+
+.Indicator {
+  cursor: pointer;
+  border-radius: 4px;
+  box-shadow: 0 6px 24px 0 #0000001a;
+}
+
+.Indicator.Warning {
+  border: 1px solid #e8b923;
+}
+
+.Indicator.Error {
+  border: 1px solid #f7a4a3;
+}
```

Comment:
These four doesn't need to be themeable as it's not user-facing, but I'll try to use `color-mix` here and see if I can replace these.